travis-5.x.cfg

```diff
@@ -1,7 +1,7 @@
 [buildout]
 extends = test-5.x.cfg buildout-cache.cfg
 parts = download install test
-plone-series = 5.1
+plone-series = 5.2
```

Comment:
Still have problems:

```bash
Installing download.
Downloading https://launchpad.net/plone/5.2/5.2.0/+download/Plone-5.2.0-UnifiedInstaller.tgz
While:
  Installing download.
Error: Error downloading extends for URL https://launchpad.net/plone/5.2/5.2.0/+download/Plone-5.2.0-UnifiedInstaller.tgz: HTTP Error 404: Not Found
```

The UnifiedInstaller url of Plone 5.2 in launchpad is:

https://launchpad.net/plone/5.2/5.2/+download/Plone-5.2.0-UnifiedInstaller.tgz

But download part tries to download:

https://launchpad.net/plone/5.2/5.2.0/+download/Plone-5.2.0-UnifiedInstaller.tgz

This is because download part  uses [${versions:Plone}](https://github.com/collective/buildout.plonetest/blob/master/buildout-cache.cfg#L20) to set the version, but the Plone version at:

https://dist.plone.org/release/5-latest/versions.cfg

is `5.2.0` and not `5.2`. 

This also occurs in `travis-5.2.x.cfg`.

This probably not will occur when Plone 5.2.1 is released.

I still think this file should be empty, only with a deprecated warning.